tests/phpunit/tests/functions/forceSslAdmin.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Tests for the force_ssl_admin function.
+ *
+ * @group functions.php
+ *
+ * @covers ::force_ssl_admin
+ */#
+class Tests_Functions_ForceSslAdmin extends WP_UnitTestCase {
+
+	/**
+	 * @ticket 57261
+	 */
+	public function test_force_ssl_admin() {
+		$default = force_ssl_admin();
+		$this->assertFalse( $default, 'default' );
+		$this->assertFalse( force_ssl_admin( true ), 'set true' );
+		$this->assertTrue( force_ssl_admin(), 'check is still true' );
+
+		// reset to default
+		$this->assertTrue( force_ssl_admin( $default ), 'set false and back to default' );
```

Comment:
This reset will not happen if any of the previous assertions fail.

Suggest storing each result in a variable, resetting the value to `$default`, then performing assertions on each variable.

i.e.

```php
$default = force_ssl_admin();
$before_setting_to_true = force_ssl_admin( true );
$after_setting_to_true = force_ssl_admin();
$before_reset = force_ssl_admin( $default );

$this->assertSame( $before_setting_to_true, $default, ... );
$this->assertTrue( ..., '' );
$this->assertTrue( ..., '' );
$this->assertTrue( ..., '' );
$this->assertSame( force_ssl_admin(), $default, 'force_ssl_admin() was not reset.' );
```

Reply:
done